dfdaemon/handler/root_handler.go

```diff
@@ -18,16 +18,15 @@ package handler
 
 import (
 	"net/http"
-	"net/http/pprof"
+	// pprof will inject handlers for users to profile this program
+	_ "net/http/pprof"
 )
 
-// New returns a new handler for dfdaemon
+// New returns a new http mux for dfdaemon
 func New() *http.ServeMux {
-	s := http.NewServeMux()
+	s := http.DefaultServeMux
 	s.HandleFunc("/args", getArgs)
-	s.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
-	s.HandleFunc("/debug/pprof/", pprof.Index)
-	s.HandleFunc("/debug/version", getVersion)
 	s.HandleFunc("/env", getEnv)
+	s.HandleFunc("/version", getVersion)
```

Comment:
/debug/version? We should guarantee backward compatibility.

Reply:
Currently I am testing these routers in master branch' s code. I found when I open localhost:65001/debug/version in the browser, it will redirect to my registry, currently dockerhub. Is it a bug? Can you reproduce this in your environment? @Starnop